assets/src/scss/layout/_navbar.scss

```diff
@@ -208,6 +208,14 @@ a.nav-link {
   width: 100%;
   text-decoration: none !important;
 
+  span {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    line-clamp: 3;
+    -webkit-line-clamp: 2;
```

Comment:
Shouldn't we use the same value for both properties?

Reply:
Absolutely, yes 🤦‍♀️

Reply:
Updated!